lib/build.js

```diff
@@ -21,6 +21,7 @@ exports.usage = 'Invokes `' + (win ? 'msbuild' : 'make') + '` and builds the mod
 function build (gyp, argv, callback) {
   var release = processRelease(argv, gyp, process.version, process.release)
     , makeCommand = gyp.opts.make || process.env.MAKE
+      || (process.platform.indexOf('aix') != -1 && 'gmake')
```

Comment:
Maybe platform-specific overrides are better done in parseArgv / parseOpts in lib/node-gyp.js.